charts/stable/emulatorjs/questions.yaml

```diff
@@ -1,7 +1,14 @@
 # Include{groups}
 portals:
-  open:
+  admin:
 # Include{portalLink}
+  open:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$variable-service.front.ports.front.port"
```

Comment:
This won't work if ingress for `front` is enabled.
As it will try to open front.domain.com:10xxx

Reply:
There isn't currently a way to automagically change this on non-main services

Reply:
Dang i was referencing [this](https://github.com/truecharts/charts/blob/master/charts/stable/ubooquity/questions.yaml) and didn't see the single ingress.

Reply:
Well That’s gonna have the same issue anyway. We can merge this one too. But will only work with load balancer type

Reply:
Actually no, it will only work without ingress. As the url will change based on ingress

(That’s true for both apps)

Reply:
Yea i test it with ingress... had to remove the port. Maybe if we can validate that if ingress is enabled the port switches to 443 instead of the default.

Reply:
There are special override for values.yaml.
This cannot be edited in question.yaml